src/Components/Main_Video/Main_Video.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Main_Video_Style.css';
+
+function Main_Video(props) {
+    return (
+        <div className="video">
+            {
+                props.video.map((res, index) => {
+                    return (
```

Comment:
tab spacing is more in some of the files like this one. Change the tab space in vscode to align properly.